index.html

```diff
@@ -1,216 +0,0 @@
-    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
-    <link href="https://fonts.googleapis.com/css2?family=Encode+Sans:wght@100;300;500;600;800;900&family=Poppins:wght@100;200;300&family=Righteous&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Encode+Sans:wght@200;400;600;800;900&family=Poppins:wght@100;200;300&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
-    <title>driveneats.com</title>
-  </head>
-
-  <!-----------------------------BODY---------------------------------->
-
-  <body>
-
-    <!---------------------------BARRA TOPO--------------------------------->
-
-    <div class="barratop">
-        <h1 class="nomeapp">DrivenEats</h1>
-        <h2 class="slogan">Sua comida em 6 minutos</h2>
-    </div>
-
-    <!--------------------------COMIDAS-------------------->
-
-    <h1 class="primeirocespaço">Primeiro, seu prato</h1>
-
-    <div class="prato" data-identifier="dishes">
-
-      <div id="salada" class="umprato" onclick="escolherPratoSalada()" data-identifier="food-option">
-
-        <img class="imagem" src="./images/frango_yin_yang 1.png">
-        <ion-icon id="saladacheck" name="checkmark-circle-outline"></ion-icon>
-        <div class="texto">
-          <h1>Frango Yin Yang</h1>
-          <h2>Um pouco de batata, um pouco de salada</h2>
-          <h3>R$ 14,90</h3>
-        </div>
-
-      </div>
-
-      <div class="espaço"> </div>
-
-      <div id="feijao" class="umprato" onclick="escolherPratoFeijao()" data-identifier="food-option">
-
-        <img class="imagem" src="./images/frango_yin_yang 1.png">
-        <ion-icon id = "feijaocheck" name="checkmark-circle-outline"></ion-icon>
-        <div class="texto">
-          <h1>Frango Yin Yang</h1>
-          <h2>Um pouco de batata, um pouco de feijão</h2>
-          <h3>R$ 13,90</h3>
-        </div>
-
-      </div>
-
-      <div class="espaço"></div>
-
-      <div id="arroz" class="umprato" onclick="escolherPratoArroz()" data-identifier="food-option">
-
-        <img class="imagem" src="./images/frango_yin_yang 1.png" />
-        <ion-icon id = "arrozcheck" name="checkmark-circle-outline"></ion-icon>
-        <div class="texto">
-          <h1>Frango Yin Yang</h1>
-          <h2>Um pouco de batata, um pouco de arroz</h2>
-          <h3>R$ 12,90</h3>
-        </div>
-
-      </div>
-
-      <div class="espaço"></div>
-
-    </div>
-
-    <h1 class="primeiro">Agora, sua bebida</h1>
-
-    <div class="prato" data-identifier="drinks">
-
-      <div id = "coca" class="umprato" onclick="escolherBebidaCoca()" data-identifier="food-option">
-
-        <img class="imagem" src="./images/coquinha_gelada 1.png"/>
-        <ion-icon id = "cocacheck" name="checkmark-circle-outline"></ion-icon>
-        <div class="texto">
-          <h1>Coca</h1>
-          <h2>600ml</h2>
-          <h3>R$ 7,90</h3>
-        </div>
```

Comment:
`texto`? Que texto é esse? Onde vive? Do que se alimenta?

lembrar de colocar nomes mais semanticos nas classes, assim quando algum outro programador for olha no seu codigo ele vai conseguir identificar de cara do que se trata.

Uma opção, nesse caso, seria trocar `texto` para `informacao-pedido`, ou em ingles `order-info'